docs/blog/newsletter-june-2018.rst

```diff
@@ -0,0 +1,122 @@
+
+If you're interested in a more virtual way to engage with the community, you could also think about volunteering to help out with the year-round organization. When we come back in August, we'll be putting out a call for volunteers to help both with the newsletter team as well as a few other working groups. So have a think about whether and how you might want to pitch in.
+
+One other quick thing to note was an interesting tidbit from the `#meta <https://writethedocs.slack.com/messages/meta>`_ channel about the pros and cons of Slack's message limit. Since it's a topic that comes up on a relatively regular basis, we thought it'd be worth addressing here. Essentially, while it can be frustrating to see valuable conversations and resources swallowed by the void, the message limit helps foster a more comfortable space for community conversation, without the looming knowledge that every word you write will be preserved, publicly, for all time.
+
+We do try to offset the transience of the Slack discourse at large with this here newsletter, where we capture a handful of pithy conversations each month. Speaking of which, let's get to em!
+
+---------------------------------------------
+Helping your project managers understand docs
+---------------------------------------------
+
+Many of us have been in the debatably enviable position of being a company's first docs person. In such situations, it's critical to get your teammates, especially project managers and the like, to understand both what you do and what you need to be able to work effectively.
+
+So in this situation, someone asked on Slack this month, what are the top things you'd want to communicate to your PM to help them understand your role?
+
+The crowd came back with some great suggestions, as usual:
+
+* **Process definition.** Docs processes are often new to non-docs folks, so they're a useful place to start. Share how new content gets defined, created, and reviewed; how ongoing changes are batched and implemented; how your publishing workflow works, step-by-step.
+
+* **Content = product.** The sooner you can get your team to start thinking about the docs as *part of the product* and not as some separate thing, the better. It will make it much easier to integrate your docs cycle with your product cycle.
+
+* **Getting in on the planning.** Speaking of which, impress upon your team the value of involving you at the planning stages of the cycle. So much of what docs folks can contribute to a product happens long before anything gets built. Plus, being present for the scoping conversations is necessary if you want to be able to manage your time.
+
+* **Timeline management.** Explain that docs take time. If your ship date is the end of week six, that means there needs to be a (mostly-)working version available by, say, the Tuesday of week four... not the Thursday of week six. Being explicit about your scheduling expectations can save you a lot of future headaches.
+
```

Comment:
no hyphen after "mostly" and add space after close-parenthesis

...needs to be a (mostly) working...